bindings_ffi/README.md

```diff
@@ -38,19 +38,36 @@ You'll need to do the following one-time setup to run Kotlin tests:
 
 If you want to skip the setup, you can also run `cargo test -- --skip kts` to only run Rust unit tests. CI will run all tests regardless.
 
+# Debugging
```

Comment:
We should also add some notes around the expected `docker-compose.yml` format to get a working local backend, and warn users that this won't work with the dev network just yet.

Reply:
Done - added a note. Haven't updated the Android example app to use localhost, will do that in next PR